promql/engine_test.go

```diff
@@ -35,13 +33,17 @@ import (
 	"github.com/prometheus/prometheus/model/timestamp"
 	"github.com/prometheus/prometheus/promql/parser"
 	"github.com/prometheus/prometheus/storage"
+	"github.com/prometheus/prometheus/tsdb/tsdbutil"
 	"github.com/prometheus/prometheus/util/stats"
+	"github.com/prometheus/prometheus/util/teststorage"
 )
 
 func TestMain(m *testing.M) {
 	goleak.VerifyTestMain(m)
 }
 
+var ctxbg = context.Background()
```

Comment:
What's this? That's unusual -- it's fine to do this add-hoc, why package lvl var?

Reply:
Ah there were so many `test.Context()`s to be replaced that i thought that would be a good idea! Shall i inline it?

Reply:
inlined it and got rid of the variable!